ui/lib/kmip/addon/components/edit-form-kmip-role.js

```diff
@@ -1,55 +1,30 @@
 import EditForm from 'core/components/edit-form';
 import layout from '../templates/components/edit-form-kmip-role';
-import { Promise } from 'rsvp';
 
 export default EditForm.extend({
   layout,
-  display: null,
+  model: null,
+
   init() {
     this._super(...arguments);
-    let display = 'operationAll';
-    if (this.model.operationNone) {
-      display = 'operationNone';
-    }
-    if (!this.model.isNew && !this.model.operationNone && !this.model.operationAll) {
-      display = 'choose';
+
+    if (this.model.isNew) {
+      this.model.set('operationAll', true);
     }
-    this.set('display', display);
   },
 
   actions: {
-    updateModel(val) {
-      // here we only want to toggle operation(None|All) because we don't want to clear the other options in
-      // the case where the user clicks back to "choose" before saving
-      if (val === 'operationAll') {
-        this.model.set('operationNone', false);
-        this.model.set('operationAll', true);
-      }
-      if (val === 'operationNone') {
-        this.model.set('operationNone', true);
-        this.model.set('operationAll', false);
-      }
+    switchUpdated(checked) {
```

Comment:
this is a nit, but i think `toggleOperations`, `toggleOperationNone` or some variant of that would be more clear

Reply:
nits are good! updated to in cb43b5c to `toggleOperationSpecial`